games-emulation/dosbox-x/dosbox_x-2022.12.26.recipe

```diff
@@ -0,0 +1,78 @@
+SUMMARY="A DOS emulator for running DOS applications and games"
+DESCRIPTION="DOSBox-X is an open-source DOS emulator for running \
+DOS applications and games. DOS-based Windows such as Windows 3.x \
+and Windows 9x are officially supported. Compared to DOSBox, DOSBox-X \
+is much more flexible and provides more features."
+HOMEPAGE="https://dosbox-x.com/"
+COPYRIGHT="2002-2023 The DOSBox-X Team"
+LICENSE="GNU GPL v2"
+REVISION="1"
+upstreamVersion="2022.12.26"
+SOURCE_URI="https://codeload.github.com/joncampbell123/dosbox-x/zip/dc350a51928bfede9722b59d40f9ef8d39bf93d0"
+SOURCE_DIR="dosbox-x-dc350a51928bfede9722b59d40f9ef8d39bf93d0"
+CHECKSUM_SHA256="7cc8ef45c028e91d2ba69fd1885f910db81f9b881776a139c8ba8f7007ecad83"
+ADDITIONAL_FILES="dosbox.rdef"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	dosbox_x$secondaryArchSuffix = $portVersion
+	apps:dosbox_x = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libglu$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	lib:libncurses$secondaryArchSuffix
+	cmd:kdialog
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libGL$secondaryArchSuffix
+	devel:libglu$secondaryArchSuffix
+	devel:libpng16$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	devel:libncurses$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:find
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:make
+	"
```

Comment:
These entries in the REQUIRES/PROVIDES should be sorted alphabetically(after the haiku*) entry which is always first.

Reply:
Done, they should now be sorted